Source/AcceleratorLattice/AcceleratorLattice.cpp

```diff
@@ -35,6 +41,12 @@ AcceleratorLattice::ReadLattice (std::string const & root_name, amrex::ParticleR
         m_lattice_defined = true;
     }
 
+    bool reverse = false;
+    pp_lattice.queryAdd("reverse", reverse);
+
+    if (reverse)
+        std::reverse(lattice_elements.begin(), lattice_elements.end());
```

Comment:
Here's my pickiness - can you wrap this in braces `if () {}`?

Reply:
no prob, done!